ptp/components/mixins/word_mappings.py

```diff
@@ -72,6 +72,9 @@ def __init__(self): #, name, class_type, config):
             if word != '' and word not in self.word_to_ix:
                 self.word_to_ix[word] = len(self.word_to_ix)
 
+        if self.config["export_pad_mapping_to_globals"]:
+            self.globals["pad_mapping"] = self.word_to_ix['<PAD>']
```

Comment:
Please rename pad_mapping to pad_index and export_pad_mapping to export_pad_index

then when one will see
ignore_index: pad_index
then it will make more sense (comparing "apples to apples")